src/gov/usgs/earthquake/product/AbstractListener.java

```diff
@@ -306,6 +329,12 @@ public void configure(final Config config) throws Exception {
 		} else {
 			includeDevelopments = DEFAULT_INCLUDE_SCENARIOS;
 		}
+		
+		includeFlag = config.getProperty(INCLUDE_ACTUALS_PROPERTY);
+		if (includeFlag != null) {
+			includeActuals = (includeFlag.equalsIgnoreCase("no") ||
```

Comment:
this seems backwards, when includeActuals configured as "no" or "false" it should be false instead of true.